src/components/navbar/navbar.tsx

```diff
@@ -28,14 +31,19 @@ const Navbar = ({ logo }: NavbarProp) => {
             <ul className="navbar-nav my-2 my-lg-0">
               {navItemsList.map((navItem, key) => {
                 return (
-                  <NavItem
-                    key={key}
-                    name={navItem.name}
-                    link={navItem.link}
-                    id={navItem.id}
-                    isButton={navItem.isButton}
-                    openInNewTab={navItem.openInNewTab}
-                  />
+                  <React.Fragment key={key}>{
+                    navItem.id === 'search-quote-btn' ? (
+                      <SearchPage />
+                    ) : (
+                      <NavItem
+                        name={navItem.name}
+                        link={navItem.link}
+                        id={navItem.id}
+                        isButton={navItem.isButton}
+                        openInNewTab={navItem.openInNewTab}
+                      />
+                    )}
+                   </React.Fragment>
                 );
```

Comment:
## Feedback

- use camel case for naming folder and files
- remove unwanted comments and codes from code
- it is generally not recommended to modify the project config files such as `package.json` until and unless absolutely required

Reply:
Noted.